config/settings.yml

```diff
@@ -11,3 +11,5 @@ gemfile_checker:
   pr_contacts: []
 issue_manager:
   repo_names: []
+label_notifier:
+  labelname: []
```

Comment:
1. please change to have an underscore between 2 words (ruby style)
2. switch to plural? (label_names)